java/google/registry/rde/imports/RdeHostImportAction.java

```diff
@@ -139,14 +139,8 @@ public void vrun() {
       } catch (Exception e) {
```

Comment:
It's not letting me add a comment on line 137, but can you change the "hosts skipped" counter to be something a little bit more descriptive?  "Skipped" doesn't tell us why it was skipped.  How about "already existing hosts skipped"?